### PR TITLE
Update docs based on tech writer feedback

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
- Update docs based on tech writer feedback

This PR updates the Getting Started page as well as all guides based on initial feedback from tech writers.

### CODE REVIEW

**Comment by gsheni** [comment]
Overall, it looks good, and should be ready to merge after the 1 change.

File: docs/source/start.ipynb [hunk 1]
```diff
@@ -112,7 +112,7 @@
    "source": [
     "## Selecting Columns\n",
     "\n",
-    "Now that we have logical types we are happy with, we can select a subset of the columns based on their logical types. Let's select only the columns that have a logical type of ``Integer`` or ``Double``:"
+    "Now that you've prepared logical types, you can select a subset of the columns based on their logical types. Select only the columns that have a logical type of `WholeNumber` or `Double`."
```
[the remark this answers is not in the document]
Comment (tamargrey) [reply docs/source/start.ipynb:115]: We got rid of `WholeNumber`, not `Integer`, right?

[the remark this answers is not in the document]
Comment (thehomebrewnerd) [reply docs/source/start.ipynb:115]: Yeah, we dropped WholeNumber. Updating this now to be consistent with the code that follows - got bit by a copy/paste here...

[the remark this answers is not in the document]
Comment (gsheni) [reply docs/source/start.ipynb]: Woops, sorry my comment was wrong, meant to reference WholeNumber.

**Review by gsheni — APPROVED** [review]
lgtm

**Pull request merged**